lib/plugins/wallet/bitcoind/bitcoind.js

```diff
@@ -17,21 +17,49 @@ function fetch (method, params) {
   return jsonRpc.fetch(rpcConfig, method, params)
 }
 
+function errorHandle (e) {
+  const err = JSON.parse(e.message)
+  switch (err.code) {
+    case -4:
+      return loadWallet()
+    case -5:
+      return logger.error(`${err}`)
+    case -6:
+      throw new E.InsufficientFundsError()
+    case -18:
+      return createWallet()
+    default:
+      throw e
+  }
+}
```

Comment:
Won't this return unexpected results? Let's say we're running `sendCoins` and there's no wallet, when handling the error we will return the `createWallet` output, right?